src/pages/organisationUnits/New.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { FormBase } from '../../components'
+import { SECTIONS_MAP, useOnSubmitNew, validate } from '../../lib'
+import {
+    FormValues,
+    initialValues,
+    OrganisationUnitFormField,
+    organisationUnitSchema,
+} from './form'
+import { DefaultNewFormContents } from '../../components/form/DefaultFormContents'
+
+const formatFormValues: (values: FormValues) => Record<string, unknown> = (
+    values
+) => {
+    return {
+        name: values.name,
```

Comment:
I think we can simplify this a bit, at least spreading the properties we're not changing.
Maybe we could create a composite-field-component `GeometryFields`, that renders both `longitude` and `latitude` and handle the formatting there?

Reply:
We could also do `organisationUnitSchema.parse(values)`, and then do the transformation we want?

Reply:
i've refactor this (very simple change) to be less morbose. Have a look, I'm not super convinced it's worth refactoring the geomeryfield as that formatting is something i believe should stay as far from the core model as possible, and also feels a bit  of a complication as for now it's only used here. For attributeValues i agree since it's used everywhere we can come up with something better - let's talk about this :)